src/Compression/CompressionCodecZSTDQAT.cpp

```diff
@@ -0,0 +1,121 @@
+#ifdef ENABLE_ZSTD_QAT_CODEC
```

Comment:
Another small thing: It would be cool if ENABLE_ZSTD_QAT_CODEC could show up in system table `system.build_options`.

Reply:
Done